docs/admin/federation/kubefed.md

```diff
@@ -0,0 +1,168 @@
+---
+assignees:
+- madhusudancs
+
+---
+Kubernetes v1.5 introduced a new command line tool called `kubefed`
+to facilitate the administration of cluster federations. `kubefed`
+stands for "Kubernetes Federate". It aids in both deploying a
```

Comment:
@nikhiljindal we wanted to choose a verb, like `kubectl` = "Kubernetes Control".